frontend/src/components/Utils/utils.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Format a timestamp (in milliseconds) into a pretty string with just the time.
+ * @param {int} msTimestamp 
```

Comment:
nit: Add new line before @param.
```suggestion
 *
 * @param {int} msTimestamp 
```